task_processing/runners/subscription.py

```diff
@@ -33,7 +33,7 @@ def run(self, task_config):
         return self.executor.run(task_config)
 
     def kill(self, task_id):
-        return self.executor.kill(task_id)
+        self.executor.kill(task_id)
```

Comment:
why don't we want to return here?

Reply:
kill is not synchronous, so it should not return anything.

Reply:
i prefer everything returning and everything having a value. less things to fix when implementation changes.